audiorecorder

```diff
@@ -110,10 +110,15 @@ _metadata_gui(){
     cb.label = Cancel
     "
 
-    pashua_configfile=$(/usr/bin/mktemp /tmp/pashua_XXXXXXXXX)
-    echo "${gui_conf}" > "${pashua_configfile}"
-    pashua_run
-    rm "${pashua_configfile}"
+    if  [[ $OSTYPE = darwin* ]] ; then
```

Comment:
What about editing the configuration file with Nano, if Pashua not there, instead of if not macOS?

I guess – but possibly I’m wrong – that at the installation it just says that Pashua was not installed, if it’s the case, but does not stop the installation of audiorecorder. You could use the Nano route in this specific cas too.